packages/core/data-transfer/lib/providers/local-file-source-provider.ts

```diff
@@ -59,17 +61,13 @@ class LocalFileSourceProvider implements ISourceProvider {
   }
 
   /**
-   * Pre flight checks regarding the provided options (making sure that the provided path is correct, etc...)
+   * Pre flight checks regarding the provided options, opening the file
    */
-  bootstrap() {
-    const { path } = this.options.file;
-    const isValidBackupPath = fs.existsSync(path);
-
-    // Check if the provided path exists
-    if (!isValidBackupPath) {
-      throw new Error(
-        `Invalid backup file path provided. "${path}" does not exist on the filesystem.`
-      );
+  async bootstrap() {
+    try {
+      this.#filestream = createReadStream(this.options.file.path);
```

Comment:
Since we're dealing with a lot of stream in our context. Don't you think it'll make it harder to understand it's a fs.ReadStream here?

Reply:
Also, why do we open the file only once now? Won't it close after the first stage and not be usable anymore?